src/core/xds/grpc/xds_certificate_provider.h

```diff
@@ -104,6 +108,9 @@ class XdsCertificateProvider final : public grpc_tls_certificate_provider {
   std::string identity_cert_name_;
   std::vector<StringMatcher> san_matchers_;
   bool require_client_certificate_ = false;
+  std::string sni_;
+  bool auto_host_sni_;
+  bool auto_sni_san_validation_ = false;
```

Comment:
Don't need `= false` here, since the ctor is always initializing it.

Reply:
There are two constructors, and only one of them initializes it.